UnityAds/src/main/java/com/mopub/mobileads/UnityAdsAdapterConfiguration.java

```diff
@@ -62,32 +62,33 @@ public void initializeNetwork(@NonNull final Context context, @Nullable final Ma
         Preconditions.checkNotNull(context);
         Preconditions.checkNotNull(listener);
 
-        boolean networkInitializationSucceeded = false;
-
         synchronized (UnityAdsAdapterConfiguration.class) {
             try {
                 if (UnityAds.isInitialized()) {
-                    networkInitializationSucceeded = true;
-                } else if (configuration != null && context instanceof Activity) {
-                    UnityRouter.initUnityAds(configuration, (Activity) context);
-
-                    networkInitializationSucceeded = true;
+                    listener.onNetworkInitializationFinished(UnityAdsAdapterConfiguration.class, MoPubErrorCode.ADAPTER_INITIALIZATION_SUCCESS);
+                } else if (configuration != null && context != null) {
```

Comment:
return from the end of if here. and doesn't need the else if.

Reply:
@jindoujiao just to rewrite this block but not remove the configuration or context check right ?